support-frontend/assets/pages/[countryGroupId]/student/StudentLandingPage.tsx

```diff
@@ -21,20 +21,25 @@ import { type GeoId, getGeoIdConfig } from 'pages/geoIdConfig';
 import { AccordionFAQ } from '../components/accordionFAQ';
 import StudentHeader from './components/StudentHeader';
 import { StudentTsAndCs } from './components/studentTsAndCs';
+import type { StudentDiscount } from './helpers/discountDetails';
 import { getStudentFAQs } from './helpers/studentFAQs';
 import { getStudentTsAndCs } from './helpers/studentTsAndCsCopy';
 
+type StudentLandingPageProps = {
+	geoId: GeoId;
+	productKey: ActiveProductKey;
+	ratePlanKey: ActiveRatePlanKey;
+	landingPageVariant: LandingPageVariant;
+	studentDiscount?: StudentDiscount;
```

Comment:
Should `studentDiscount` be required here? Won't there always be one? (Or is this the case where the promo code for UTS isn't provided?)

Reply:
Your right, you should always get a `studentDiscount` even for a UTS student. It should not be optional (that check will now be applied higher up in the `StudentLandingPageContainer`. Updated to reflect the change.